src/Features/Core/Portable/InitializeParameter/AbstractAddParameterCheckCodeRefactoringProvider.cs

```diff
@@ -518,22 +520,30 @@ private static TStatementSyntax CreateNullCheckStatement(SemanticModel semanticM
             return generator.TypeExpression(typeSymbol);
         }
 
-        private static SyntaxNode CreateArgumentNullException(
+        private SyntaxNode CreateArgumentNullException(
             Compilation compilation, SyntaxGenerator generator, IParameterSymbol parameter)
         {
             return generator.ObjectCreationExpression(
                 GetTypeNode(compilation, generator, typeof(ArgumentNullException)),
                 generator.NameOfExpression(generator.IdentifierName(parameter.Name)));
         }
 
-        private static SyntaxNode CreateArgumentException(
+        private SyntaxNode CreateArgumentException(
             Compilation compilation, SyntaxGenerator generator, IParameterSymbol parameter)
         {
-            // Note "message" is not localized.  It is the name of the first parameter of 
-            // "ArgumentException"
+            var content = new List<SyntaxNode>()
+            {
+                generator.InterpolatedStringText(generator.InterpolatedStringTextToken($"'{{{generator.NameOfExpression(generator.IdentifierName(parameter.Name))}}}'")),
+                generator.InterpolatedStringText(generator.InterpolatedStringTextToken("cannot be null or empty"))
+            };
             return generator.ObjectCreationExpression(
                 GetTypeNode(compilation, generator, typeof(ArgumentException)),
-                generator.LiteralExpression("message"),
+                generator.InterpolatedStringExpression
+                (
+                    CreateInterpolatedStringStartToken().WithLeadingTrivia(content.First().GetLeadingTrivia()),
+                    content,
+                    CreateInterpolatedStringEndToken().WithTrailingTrivia(content.Last().GetTrailingTrivia())
+                ),
```

Comment:
non-idiomatic formatting for this layer.

Reply:
need to change
`content.First()`
to
`content[0]`
for example?

Reply:
no, this is about the formatting, we don't break invocations up like this where the open paren and close paren are on different lines.

Reply:
i understood)